### PR TITLE
chore: replace `clean` transaction with sequence of promises, increased `beforeEach` timeout

### PR DESCRIPTION
**Pull request by jkomyno**
Attempt to unlock CI.

### CODE REVIEW

**Comment by janpio** [comment]
(Now better no merges from `main` any more as that disables these tests)

**Comment by Jolg42** [comment]
Note: the test is skipped on main since https://github.com/prisma/prisma/pull/15565 because it was blocking the release.
After the release: We need to continue this PR here to make the test more robust and unskip it.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -187,7 +187,7 @@ jobs:
 
       # If we detected (in detect_jobs_to_run) that only functional test files changed
       # We only run the tests that changed (or were added)
-      - run: pnpm run test:functional --onlyChanged
+      - run: pnpm run test:functional --changedSince=main
```
Comment (Jolg42) [inline .github/workflows/test.yml:190]:
```suggestion
      - run: pnpm run test:functional --onlyChanged
```

**Comment by Jolg42** [comment]
Sorry for the mess here, everything is back to normal.

Note I created a PR to checl that only unskip the test to see the current status and it fails in Buildkite https://github.com/prisma/prisma/pull/15575

**Comment by jkomyno** [comment]
Let’s consider that most likely increasing timeouts in those tests is just a temporary patch for a broader issue elsewhere: too many concurrent Prisma clients are competing for resources, resulting in transactions (e.g., the one used for cleaning the db tables) taking ages

**Review by Jolg42 — APPROVED** [review]
Did a rerun and it works! let's merge ✅

**Pull request merged**